src/store/modules/ADempiere/form/VPOS/index.js

```diff
@@ -187,6 +198,46 @@ export default {
         resolve(getPointOfSales)
       })
     },
+    /**
+     * list Command Shortcut Current POS
+     */
+    listCommandShortcutsVPOS({
+      commit,
+      getters
+    }, {
+      pointOfSales
+    }) {
+      return new Promise(resolve => {
+        if (isEmptyValue(pointOfSales)) {
+          pointOfSales = getters.getVPOS
+        }
+        const { id } = pointOfSales
+        listCommandShortcut({
+          posId: id
+        })
+          .then(response => {
+            console.log({ ...response })
```

Comment:
please remove this console.log

Reply:
remove console    https://github.com/solop-develop/frontend-core/pull/1804/commits/1197d3806a38ed49944926e0d1762c2b4c7c2afe